src/coreclr/nativeaot/docs/compiling.md

```diff
@@ -74,9 +74,28 @@ You can use this feature by adding the `StaticICULinking` property to your proje
 ```xml
 <PropertyGroup>
   <StaticICULinking>true</StaticICULinking>
+
+  <!-- Optional: Embed ICU data. Update path to match your ICU version and variant:
```

Comment:
I think this can use a better description for why one may need to do this.

I assume that dealing with the .dat file is not really optional if one wants the binary to work on a system when libicu libraries are not installed. Is that correct? If you do not embed the ICU data, where does one need to copy the ICU file to make things work?